content/docs/gke/private-clusters.md

```diff
@@ -231,13 +226,21 @@ export PROJECT_NUMBER=$(gcloud projects describe kubeflow-dev --format='value(pr
     * This is needed because your GKE nodes won't be able to pull images from non GCR
       registries because they don't have public internet addresses
 
+
+    * gcloud may return an error even though the job is
+      submited successfully and will run successfully
+      see [kubeflow/kubeflow#3105](https://github.com/kubeflow/kubeflow/issues/3105)
+
+    * You can use the Cloud console to monitor your GCB job.
+
+
 1. Follow the [instructions](https://www.kubeflow.org/docs/gke/deploy/oauth-setup/) for creating an OAuth client
 
 1. Create environment variables for IAP OAuth access
 
     ```bash
-    export CLIENT_ID=<CLIENT_ID from OAuth page>
-    export CLIENT_SECRET=<CLIENT_SECRET from OAuth page>
+     export CLIENT_ID=<CLIENT_ID from OAuth page>
```

Comment:
There seems to be too much space in front of these code lines in the rendered preview, now that the indentation is increased.